include/arch/arc/v2/embarc_mli.ld

```diff
@@ -0,0 +1,102 @@
+#ifndef EMBARC_MLI_ROM_REGION
```

Comment:
This looks like a stuff specific to ARC MLI only. I suppose that it shouldn't be in `include/arch/arc` which contains ARC architecture-specific code.

Reply:
Move it to `zephyr/modules/embarc_mli/`